### ISSUE TITLE
why the code doesnot release memory after Validation?

### ISSUE THREAD

**Issue by Rainyfish** [xinntao/ESRGAN#38]
@xinntao Hi, when the code do its first Validation, it will allocate about 2GB GPU memory, but do not release them after Validation.

**Comment by xinntao** [comment]
Did you mean the memory is not freed after the validation is over?

**Comment by Rainyfish** [comment]
@xinntao Yes, when the second validation begin, the gpu memory  occupation does not increase.
